src/modules/mount/main.py

```diff
@@ -307,6 +307,12 @@ def mount_partition(root_mount_point, partition, partitions, mount_options, moun
                                         mount_option) != 0:
                 libcalamares.utils.warning("Cannot mount {}".format(device))
 
+def enable_swap_partition(swap_device):
+    try:
+        subprocess.check_call(["swapon", swap_device])
+    except subprocess.CalledProcessError:
+        raise Exception(_("Failed to swapon " + swap_device))
```

Comment:
This exception isn't being handled.

Either create a specific exception class and handle or call `libcalamares.utils.warning()` instead of raising an exception here.  The latter would be simpler.